### PR TITLE
Update version to 20.2.0 and cleanup docs

### PR DESCRIPTION
**Pull request by onabiull**

### CODE REVIEW

File: samples/sample_common/include/version.h [hunk 1]
```diff
@@ -30,7 +30,7 @@ or https://software.intel.com/en-us/media-client-solutions-support.
 #endif
 
 #ifndef MSDK_RELEASE
-#define MSDK_TARGETAPIMINOR 27
+#define MSDK_TARGETAPIMINOR 33
```
Comment (dvrogozh) [inline samples/sample_common/include/version.h:33]: It seems we don't need this line at all if you drop this line below:

`ss << MSDK_MAJOR << "." << MSDK_MINOR << "." << MSDK_TARGETAPIMINOR << "." << MSDK_BUILD;`

And why are we doing this each time in the release? Can we do that in master permanently?

File: doc/mediasdk-man.md [hunk 2]
```diff
@@ -4150,9 +4150,6 @@ The application can attach this extended buffer to the [mfxVideoParam](#mfxVideo
 `EnableMBForceIntra` | Turn ON this option to enable usage of [mfxExtMBForceIntra](#mfxExtMBForceIntra) for AVC encoder. See the [CodingOptionValue](#CodingOptionValue) enumerator for values of this option. This parameter is valid only during initialization.
 `AdaptiveMaxFrameSize` | If this option is ON, BRC may decide a larger P or B frame size than what [MaxFrameSizeP](#MaxFrameSizeP) dictates when the scene change is detected. It may benefit the video quality. AdaptiveMaxFrameSize feature is not supported with LowPower ON or if the value of MaxFrameSizeP = 0.
 `RepartitionCheckEnable` | Controls AVC encoder attempts to predict from small partitions. Default value allows encoder to choose preferred mode, `MFX_CODINGOPTION_ON` forces encoder to favor quality, `MFX_CODINGOPTION_OFF` forces encoder to favor performance.
-`QuantScaleType` | For MPEG2 specifies mapping between quantiser_scale_code and quantiser_scale (see [QuantScaleType](#QuantScaleType) enum).
```
Comment (dvrogozh) [inline doc/mediasdk-man.md:4153]: I am kind of surprised on the amount of doc and api changes....

File: doc/mediasdk-man.md [hunk 3]
```diff
@@ -4457,32 +4452,10 @@ This structure is available since SDK API 1.1.
...
-    MFX_MVPRECISION_UNKNOWN    = 0,
-    MFX_MVPRECISION_INTEGER    = (1 << 0),
-    MFX_MVPRECISION_HALFPEL    = (1 << 1),
-    MFX_MVPRECISION_QUARTERPEL = (1 << 2)
-};
-
 typedef struct {
     mfxExtBuffer Header;
     mfxU16       FilterStrength;
-    mfxU16       Overlap;            /* tri-state option */
```
Comment (dvrogozh) [inline doc/mediasdk-man.md:4480]: hm. Why we are dropping these APIs? Aren't they in 1.33? I don't quite get the idea...

File: samples/sample_common/include/version.h (hunk 1 above)
Comment (onabiull) [reply samples/sample_common/include/version.h:33]: Will do.

File: doc/mediasdk-man.md (hunk 3 above)
Comment (onabiull) [reply doc/mediasdk-man.md:4480]: It is different part.

**Pull request merged**